experiments/targetShift.py

```diff
@@ -0,0 +1,81 @@
+from sklearn.linear_model import LinearRegression
+from doubt import Boot
+import seaborn as sns
+from doubt import QuantileRegressionForest as QRF
+
+import pandas as pd
+import os
+import matplotlib.pyplot as plt
+from doubt.datasets import PowerPlant
+
+
+
+path_out = os.path.join("experiments", "results", "targetShift")
+
+datasets = {"PowerPlant": PowerPlant().split()}
+models = {"Linear": Boot(LinearRegression()),
+          'QuantileRegressor':QRF()}
+
+
+for dataset in datasets:
+    for model in models:
+        X, y = datasets[dataset]
+        clf = models[model]
+
+        # Join and order Dataframe
+        columns = [f"col_{num}" for num in range(X.shape[1])]
+        X = pd.DataFrame(X, columns=columns)
+        X["target"] = y
+        X = X.sort_values("target").reset_index(drop=True)
+
+        # Determine fraction to use in test (will be x2 the split point)
+        split_point = int(X.shape[0] * 0.35)
+
+        # Select smaller and bigger than splitting point
+        train = X.iloc[split_point : X.shape[0] - split_point]
+        pre = X.iloc[:split_point]
+        post = X.iloc[split_point:]
```

Comment:
Shouldn't this be `X.shape[0] - split_point` instead of `split_point`?

Reply:
Let me think...
Let say the dataset has 300 rows.

I want to select for training the rows 100-200
So I am doing 
[100: 300 - 100]

No? Might be wrong

Reply:
Yep, I understand what you're saying, but right now you're defining `pre` to be the first 100 and `post` to be everything after 100, i.e. the interval [100,..., 300). Instead, you want to set `post = X.iloc[X.shape[0] - split_point : ]`.

Reply:
You are right! Just corrected that!

Reply:
![QuantileRegressor_PowerPlant_full](https://user-images.githubusercontent.com/43068386/121202362-84392b00-c875-11eb-97a2-da63bc45651f.png)

Reply:
results look a bit better now